### PR TITLE
Update styles css

### PR DESCRIPTION
**Pull request by KarlaChable-Ksquare**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,16 +24,13 @@ const ctx = canvas.getContext("2d"); //to acces the drawing context
...
-let started;
-let stoped = false;
 
 /* ==============================  BUILDING THE CANVAS GRID  ============================== */
 
 // Defining the canvas size, we can modify this values if we want to resize the grid
 const resolution = 10;
-const h = (canvas.width = 200);
-const w = (canvas.height = 200);
+const w = (canvas.width = 500);
```
Comment (JacintoDelahoz-Ksquare) [inline index.js:32]: Should we remove the parenthesis?

File: index.html [hunk 2]
```diff
@@ -11,13 +11,17 @@
...
   <header>
-    <h1>The Game of Life</h1>
+    <h1 class="box">The Game of Life</h1>
     <p>Click on the grid to choose the game's initial configuration</p>
     <p>Click play and watch the game evolve</p>
     <div class="info-general">
-      <section class="info-gen">how many generations have passed?</section>
-      <section class="info-grid"> Cells Status </section>
+      <div class="generations">
+        <section class="info-gen">how many generations have passed?</section>
```
Comment (ClaudiaDiaz-Ksquare) [inline index.html:19]: Would be nice to capitalize the fisrt letter, setting it to "How" instead (:

File: index.js (hunk 1 above)
Comment (ClaudiaDiaz-Ksquare) [reply index.js:32]: We can , I tried in my local branch and in the end it's the same result

File: styles.css [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (ClaudiaDiaz-Ksquare) [inline styles.css:1]: If we're not going to use this commented code anymore, we could delete it in the next commit, what do you think?

**Review by ClaudiaDiaz-Ksquare — COMMENTED** [review]
Just remarked some typos and made suggestions to correct

**Pull request merged**